tenable/sc/scan_instances.py

```diff
@@ -258,6 +264,12 @@ def list(self, fields=None):
             params['fields'] = ','.join([self._check('field', f, str) 
                 for f in fields])
 
+        if start_time:
+            params['startTime'] = str(start_time)
+
+        if end_time:
+            params['endTime'] = str(end_time)
```

Comment:
str(end_time) should be self._check('end_time', end_time, int)

Reply:
Good call.  Fixed.